### DISCUSSION TITLE
Table structure of t_ds_task_definition

### DISCUSSION THREAD

**Discussion by LiJie20190102** [apache/dolphinscheduler#12846] [General]
The "task_execute_type" field is' 0 'by default. Should it not be empty. Because when I forget to specify the type, this field is empty, which may cause problems
![image](https://user-images.githubusercontent.com/53458004/200981193-55681915-5841-4838-bf05-3530ee340764.png)

**Comment by Tandoy** [comment]
It may be because most scenarios of Dolphin are now batch processing tasks, and only a few real-time processing tasks.
